packages/amplication-client/src/Assistant/OnboardingWithJovu.tsx

```diff
@@ -44,9 +53,16 @@ const OnboardingWithJovu: React.FC = () => {
       history,
       sendOnboardingMessage,
       setOpen,
+      trackEvent,
     ]
   );
 
+  useEffect(() => {
+    trackEvent({
+      eventName: AnalyticsEventNames.ViewOnboardingWithJovu,
+    });
+  }, []);
```

Comment:
@yuval-hazaz  you are missing the trackEvent dependency

Reply:
that's on purpose because I want this event to run only once on load, so I keep the dep list empty